shared/chat/conversation/header-area/normal/index.stories.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from 'react'
+import {action, storiesOf, createPropProvider} from '../../../../stories/storybook'
+import * as PropProviders from '../../../../stories/prop-providers'
+import {UsernameHeader, ChannelHeader} from '.'
+
+const provider = createPropProvider(PropProviders.Usernames())
+
+const defaultProps = {
+  badgeNumber: 1,
+  canOpenInfoPanel: true,
+  channelName: 'nyc',
+  infoPanelOpen: false,
+  muted: false,
+  onBack: action('onBack'),
+  onCancelPending: null,
+  onOpenFolder: action('onOpenFolder'),
+  onShowProfile: action('onShowProfile'),
+  onToggleInfoPanel: action('onToggleInfoPanel'),
+  participants: ['joshblum', 'ayoubd'],
+  smallTeam: true,
+  teamName: 'keybase',
+}
+
+const load = () => {
+  storiesOf('Chat/Header', module)
+    .addDecorator(provider)
+    .add('Username Header', () => <UsernameHeader {...defaultProps} />)
+    .add('Username Header with info panel open', () => (
+      <UsernameHeader {...defaultProps} infoPanelOpen={true} />
+    ))
+    .add('Username Header muted', () => <UsernameHeader {...defaultProps} muted={true} />)
+    .add('Username Header with onCancelPending', () => (
```

Comment:
Can you also pass in the other props that depend on `isPending`? From the connector:
```
  canOpenInfoPanel: !stateProps._isPending,
  onCancelPending: stateProps._isPending ? dispatchProps._onCancel : null,
  onOpenFolder: stateProps._isPending ? null : dispatchProps._onOpenFolder,
```